rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientRegisterActivityTest.java

```diff
@@ -44,6 +45,19 @@ public void setUp() {
                 .get();
     }
 
+    @Test
+    public void testVerifyGlobalActivityShouldBeCovidPatientRegisterActivity() throws Exception {
```

Comment:
rename to `testVerifyCurrentActivityShouldBeCovidPatientRegisterActivity`

Reply:
done